### PR TITLE
Clarifications about using hypershift

### PR DESCRIPTION
**Pull request by minkimipt**
Note about finding the right AWS availability zone

# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: use-timescale/migrate-db/hypershift.md [hunk 1]
```diff
@@ -51,6 +51,12 @@ If you're migrating from an Amazon RDS database, the best option is to run
 Hypershift on an EC2 instance in the same availability zone as your RDS
 instance.
 
+<Highlight type="important">
+Note that availability zones are assigned different names in different AWS
+accounts. To make sure that you are really using the same zone refer to [AWS
+user guide][aws-zones]
```
Comment (Loquacity) [inline use-timescale/migrate-db/hypershift.md:57]:
```suggestion
Availability zones are assigned different names in different AWS
accounts. To make sure that you are using the same zone refer to 
[AWS user guide][aws-zones]
```

File: use-timescale/migrate-db/hypershift.md [hunk 2]
```diff
@@ -51,6 +51,12 @@ If you're migrating from an Amazon RDS database, the best option is to run
 Hypershift on an EC2 instance in the same availability zone as your RDS
 instance.
 
+<Highlight type="important">
```
Comment (Loquacity) [inline use-timescale/migrate-db/hypershift.md:54]:
```suggestion
<Highlight type="note">
```

**Comment by JamesGuthrie** [comment]
@minkimipt This is useful information in general, but I'm not sure if it's applicable to users of hypershift in this context. If they are advanced enough to have multiple AWS accounts, then it might be safe to assume that they know this? If they don't have multiple AWS accounts, this doesn't apply to them at all. As we don't publish the AZ id of the service in Timescale Cloud (or do we?), they can't use this to run hypershift in the same AZ as their Timescale Cloud service either.

**Comment by Loquacity** [comment]
New Hypershift PR to come shortly 🚀

**Pull request closed without merging**